### DISCUSSION TITLE
Steps if the user has uploaded their logo to google drive and wants to display it while printing bill.

### DISCUSSION THREAD

**Discussion by Divwork92** [madrecha/jamku#912] [QA] [answered]
steps if the user has uploaded their logo to google drive and wants to display it while printing bill.

**Comment by Divwork92** [comment, marked as answer]
1. Right click on the image to get the sharable link.
2. Copy the file id from the link. For ex: If the link is like this https://drive.google.com/file/d/1k5tF13lEA4tHyrqxzcnyDv-YwIdMDIMC/view?usp=sharing , then the file id = 1k5tF13lEA4tHyrqxzcnyDv-YwIdMDIMC . Also make sure that the file is not restricted.
3. Copy the file id and paste it in this link https://drive.google.com/uc?id=fileid . So now the final link will be https://drive.google.com/uc?id=1k5tF13lEA4tHyrqxzcnyDv-YwIdMDIMC . 
4. Enter the above link into the bill print page.

**Comment by dubenikhil461** [comment]
1. Right click on the image to get the sharable link.
2. Copy the file id from the link. For ex: If the link is like this [ https://drive.google.com/file/d/1ihWXUPXNHUk14aQR2KH2aLvGvH4tO8dz/view?usp=sharing](url) , then the fileid = 1ihWXUPXNHUk14aQR2KH2aLvGvH4tO8dz . Also make sure that the file is not restricted.
3. Copy the file id and paster it in this link [https://drive.google.com/thumbnail?id=fileid&sz=w1000](url)  So now the final link will be [https://drive.google.com/thumbnail?id=1ihWXUPXNHUk14aQR2KH2aLvGvH4tO8dz&sz=w1000](url)
4. Enter the above link into the bill print page.
